dbms/src/Server/Server.cpp

```diff
@@ -201,6 +200,10 @@ namespace Debug
 extern void setServiceAddr(const std::string & addr);
 }
 
+// It's used for the initialization of Context, so we need to ensure
+// that context in Server is created after ServerInfo has set this variable.
+UInt16 max_logical_cpu_cores = 0;
```

Comment:
```suggestion
UInt16 max_logical_cpu_cores;
```

maybe no need to init?